pom.xml

```diff
@@ -343,6 +373,8 @@
             </testResource>
         </testResources>
         <plugins>
+		
+
```

Comment:
Could you please remove these extra lines?

Reply:
removed and updated in my repository